src/vcpkg/commands.build.cpp

```diff
@@ -905,7 +905,7 @@ namespace vcpkg
         auto&& scfl = action.source_control_file_and_location.value_or_exit(VCPKG_LINE_INFO);
 
         Triplet triplet = action.spec.triplet();
-        const auto& triplet_file_path = paths.get_triplet_file_path(triplet);
+        const auto triplet_file_path = paths.get_triplet_file_path(triplet);
```

Comment:
Wait this one adds a copy?

Reply:
Someone just forgot the & in the return type. It was originally `const Path`. I also fixed all of the calling code to not copy.